### PR TITLE
Create Plot Component

### PR DESCRIPTION
**Pull request by wgranger**
This PR creates a basic plot graph using the [Nivo line component](https://nivo.rocks/line/). I chose to use Nivo, as opposed to other libraries like VX or react-vis for a few reasons:

- I can import Nivo and plug in a few values to get a graph going
- Documentation is great and the tool is maintained with a good community on GitHub
- It's easy to play around with the design via the link above and see what a chart would look like

Keep in mind, this PR only gets a basic graph going with some mock data. There's still some debate over what the final graph will look like once we have an idea of how the data looks. For now, the aim of this work is to get a simple chart going and to plug it into the `MapDetail` component. You can view that component and the `Plot` component via storybook - `yarn storybook`

[InVision Design](https://projects.invisionapp.com/d/#/console/19544062/412574328/preview)

Screen Shot 2020-08-25 at 11 39 47 AM

### CODE REVIEW

**Review by beckyrother — APPROVED** [review]
🎉

File: src/components/MapDetail/components/Plot/Plot.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import { Box, Text } from 'grommet'
+import { Line, ResponsiveLine } from '@nivo/line'
```
Comment (mcbouslog) [inline src/components/MapDetail/components/Plot/Plot.js]: I don't think Line used.

File: src/components/MapDetail/components/Plot/Plot.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+	{ x: '1990', y: 72 },
+	{ x: '1995', y: 74 },
+	{ x: '2000', y: 79 },
+	{ x: '2005', y: 83 },
+	{ x: '2010', y: 77 },
+	{ x: '2015', y: 79 },
+	{ x: '2020', y: 84 },
+]
+
+const years = ['1980', '1985', '1990', '1995', '2000', '2005', '2010', '2015', '2020']
```
Comment (mcbouslog) [inline src/components/MapDetail/components/Plot/Plot.js]: I don't think this ended up being used?

File: src/components/MapDetail/components/Charts/Charts.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import { Grid } from 'grommet'
 import Chart from './Chart'
```
Comment (mcbouslog) [inline src/components/MapDetail/components/Charts/Charts.js]: Not sure if temporary, but Chart component no longer used in Charts.

File: src/components/MapDetail/components/Plot/Plot.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+					axisLeft={{
+						orient: 'left',
+						legend: 'temp',
+						legendOffset: -20,
+						legendPosition: 'middle',
+						tickSize: 0,
+						tickValues: 4
+					}}
+					yScale={{ type: 'linear', min: 'auto', max: 'auto', stacked: false }}
+					yScale={{ type: 'linear', min: 'auto', max: 'auto', stacked: false }}
```
Comment (mcbouslog) [inline src/components/MapDetail/components/Plot/Plot.js]: `yScale` duplicated

**Review by mcbouslog — APPROVED** [review]
LGTM, `nivo` looks pretty cool!

Minor comments, not blocking, up to you if want to edit this PR or include in subsequent PR.

This is silly, but my VSCode (standard) linter is throwing a bunch of warnings about tabs instead of spaces, specifically for `Plot.js` and `getAverages.js`. Definitely not blocking, but wanted to mention.

**Comment by wgranger** [comment]
Good review here. You caught a good bit of cruft I neglected to remove after getting this to work.

**Comment by wgranger** [comment]
Also, I just made the switch to VS Code from Atom recently, so my defaults were off and added 4 spaces rather than 2. I'll make the change here.

**Review by mcbouslog — APPROVED** [review]
👍

**Pull request merged**